test/MVCFramework45.FunctionalTests/project.json

```diff
@@ -14,11 +14,12 @@
         "wwwroot",
         "Views"
       ]
-    }
+    },
+    "nowarn": [ "CS0618" ]
```

Comment:
Why is the warning for using obsolete members being turned off (here an elsewhere)?